context-free.md

```diff
@@ -2,21 +2,28 @@
 
 ##  Who is this programming language for?
 
+Anyone who is interested in creating procedurally generated art.
 
 ## What is easy to do in this language? Why is it easy?
 
+Creating self similar images is very straight forward due to the way the rules call each other.
 
 ## What is hard to do in this language? Why is it hard?
 
+Controlling randomness seems a bit hard. It would be nice to do something other than just
```

Comment:
I agree, I'm not sure I understand what you're looking for - in general other than writing my own "pseudo" (and pseudo is very much the operative word) random number generator or using a known seed I'm not sure how you could control it. Or do you mean that you want the ability to scale or have some sort of weighted randomness?